components/input/src/auro-input.js

```diff
@@ -13,10 +13,12 @@ import i18n from './i18n.js';
 import BaseInput from './base-input.js';
 
 import { AuroDependencyVersioning } from '@aurodesignsystem/auro-library/scripts/runtime/dependencyTagVersioning.mjs';
+import AuroLibraryRuntimeUtils from '@aurodesignsystem/auro-library/scripts/utils/runtimeUtils.mjs';
+
 import { AuroIcon } from '@aurodesignsystem/auro-icon/src/auro-icon.js';
-import iconVersion from './iconVersion.js';
+import iconVersion from '../../../src/iconVersion.js';
```

Comment:
There is also a `components/input/src/buttonVersion.js` and `iconVersion.js`. Do we still need these files if we are pointing to the root level files?

Reply:
removed `components/input/src/*Version.js`